### PR TITLE
add closing right panel

### PR DESCRIPTION
**Pull request by vikotia**
blablabla

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -53,33 +56,75 @@ class RouteDetailsWidget extends StatelessWidget {
...
+              child: Icon(
+                  expanded ? Icons.arrow_forward_ios : Icons.arrow_back_ios),
+              onPressed: () => setState(() {
+                expanded = !expanded;
+              }),
+            ),
+          ),
+        ],
+      ),
+      floatingActionButton: FloatingActionButton(
```
Comment (dendi239) [inline lib/main.dart]: remove FloatingActionButton here, please

File: lib/main.dart [hunk 2]
```diff
@@ -36,7 +38,8 @@ class RouteCellWidget extends StatelessWidget {
         title: Wrap(
           spacing: 10,
           children: [
-            Text(place.thumbnail, style: TextStyle(fontWeight: FontWeight.bold)),
+            Text(place.thumbnail,
```
Comment (dendi239) [inline lib/main.dart:41]: Why have you changed that?

File: lib/main.dart [hunk 3]
```diff
@@ -53,33 +56,75 @@ class RouteDetailsWidget extends StatelessWidget {
...
+class MyScaffold extends StatefulWidget{
+  @override
+  _MyScaffoldState createState() => _MyScaffoldState();
+}
+
+class _MyScaffoldState extends State<MyScaffold>{
+  bool expanded = true;
+  var rightPanelWidth;
+  var buttonWidth;
+  @override
```
Comment (dendi239) [inline lib/main.dart:75]: Please, keep blank lines between functions and variables (as well as any other class's members)

File: lib/main.dart [hunk 4]
```diff
@@ -53,33 +56,75 @@ class RouteDetailsWidget extends StatelessWidget {
...
 
+class MyScaffold extends StatefulWidget{
+  @override
+  _MyScaffoldState createState() => _MyScaffoldState();
+}
+
+class _MyScaffoldState extends State<MyScaffold>{
+  bool expanded = true;
+  var rightPanelWidth;
+  var buttonWidth;
```
Comment (dendi239) [inline lib/main.dart]: Why not `double`?

File: lib/main.dart [hunk 5]
```diff
@@ -53,33 +56,75 @@ class RouteDetailsWidget extends StatelessWidget {
...
+  @override
+  Widget build(BuildContext context) {
+    final mediaQuery = MediaQuery.of(context);
+    final width = mediaQuery.size.width;
+    rightPanelWidth = max(width / 4, 228.0);
```
Comment (dendi239) [inline lib/main.dart:79]: Why `228`?

**Review by dendi239 — COMMENTED** [review]
LGTM!

**Review by dendi239 — APPROVED** [review]

File: lib/main.dart (hunk 2 above)
Comment (vikotia) [reply lib/main.dart:41]: eto ne ya, its Ctrl+Alt+L(Reformat Code)

File: lib/main.dart (hunk 5 above)
Comment (vikotia) [reply lib/main.dart:79]: I conducted an expensive high-precision study and it's the optimal size for this panel.

File: lib/main.dart (hunk 1 above)
Comment (vikotia) [reply lib/main.dart]: why

Comment (dendi239) [reply lib/main.dart]: Because it can't do anything useful

Comment (vikotia) [reply lib/main.dart]: kk

**Pull request merged**